firmware/baseband/proc_capture.cpp

```diff
@@ -27,23 +27,29 @@
 #include "utility.hpp"
 
 CaptureProcessor::CaptureProcessor() {
-    decim_0_4.configure(taps_200k_decim_0.taps, 33554432);           // to be used with decim1 (/2), then total two stages decim (/8)
-    decim_0_8.configure(taps_200k_decim_0.taps, 33554432);           // to be used with decim1 (/2), then total two stages decim (/16)
-    decim_0_8_180k.configure(taps_180k_wfm_decim_0.taps, 33554432);  // to be used alone - no additional decim1 (/2), then total single stage decim (/8)
+    decim_0_4.configure(taps_200k_decim_0.taps, 33554432);           // to be used with decim1 (/2), then total two stages decim (x8)
+    decim_0_8.configure(taps_200k_decim_0.taps, 33554432);           // to be used with decim1 (/2), then total two stages decim (x16)
```

Comment:
I thought we were using x2, x4, x8, x16, etc. to represent Oversampling (x in place of the multiplication symbol), and /2, /4, /8, /16, etc. to represent Decimation (/ for the division symbol).

Since this is Decimation, I don't understand why you changed these comments from "/" to "x".

Reply:
You are right ,  actually  ,we are mixing  3  concepts , and I think it may confuse to follow up , and that is no good ...
i-) when  ADC  can not handle low bit rates, therefore we oversampling (more than needed)  it  by "x_number" ,  and later to write it to SD card the proper needed real sample rate , we applied decimation , "/ number" . 

ii-) and in Replay list  , we got low bit rate data in those files,  and we need to upsampling or interpolate or resampling  to increase those low bit rates to be able to be transmitted by Hackrf . 

So there are  3 concepts : oversampling ("x")  for ADC capture  / interpolation or resampling ("x") for Replay   / decimation ("/") 

The problem is that we have common function talking to x8, x16,  used  to signal the oversampling  (for the ADC in Capture ) and also used to express the up-sampling or interpolation factor  (for the Replay list ) ....
Then I was trying to explain correctly ,  talking about "/" in decimation and "x·" in oversampling for ADC or up-interpolation.   but thinking about that common funcion , I stated to use just ("x") .

But you are right , then to be strick , let me change in Capture file , all the decimation comments , again to ("/")